tests/Unit/Pypp/CMakeLists.txt

```diff
@@ -10,5 +10,5 @@ add_test_library(
   "Test_Pypp"
   "Pypp"
   "${LIBRARY_SOURCES}"
-  "ErrorHandling"
+  "DataStructures;ErrorHandling"
```

Comment:
is this distinct in CMake syntax from having them on different lines? If not, why prefer same line?

Reply:
Yes because we have a function that takes a list this argument and in CMake lists are delimited by `;` (:man_shrugging:). Does that make sense?